repos/fdbt-site/src/pages/api/csvZoneUpload.ts

```diff
@@ -182,53 +188,54 @@ export const processServices = (req: NextApiRequestWithSession, formData: FileDa
 
     const errors: ErrorInfo[] = [];
     if (clickedYes && selectedServices.length === 0) {
```

Comment:
I know this code isn't part of your PR, but this last part of the function to update the session **really** shouldn't be in this function, that should be in the API.

Reply:
We should change the function so it returns 'clickedYes', as well as selectedServices. And then we can update the session in the API.